### PR TITLE
Redis Get Your S* Together

### PR DESCRIPTION
**Pull request by sancar**

### CODE REVIEW

File: data/blog/2024-06-11-redis-get-your-stream-together.mdx [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+
+<Note type="info">
+Queue Lag = Stream Lag + Number of Entries in the PEL.
+</Note>
+
+So far, so good. The implementation of this fix is really simple. The problem arose when we tried to test the new code, mostly because this time we tested more edge cases.
+
+#### miniredis
+
+We were using [miniredis](https://github.com/alicebob/miniredis), a `Pure Go Redis server for Go unit tests`. - Did I mention that we implemented the Redis spec and QStash in Go? - This library is perfect for QStash because we don't need to spin up Upstash Redis clusters to test QStash. The first small issue was that lag is not yet implemented in miniredis. It is a non-profit project with a single maintainer, so it is acceptable. As a responsible open-source contributor, I decided to add the missing feature and contribute back to open-source. https://github.com/alicebob/miniredis/pull/377
```
Comment (CahidArda) [inline data/blog/2024-06-11-redis-get-your-stream-together.mdx:58]: should we format the link at the end?
